content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md

```diff
@@ -67,6 +65,10 @@ If this process fails, retry using the iso file method above.
 
    ![SCALESidegradeReboot](/images/SCALE/SystemSettings/SidegradeRestart.png  "Reboot to Finish")
 
-8. When TrueNAS SCALE boots, you might need to [use the Shell to configure networking interfaces]({{< relref "ConsoleSetupMenuScale.md" >}}) to enable GUI accessibility.
+When TrueNAS SCALE boots, you might need to [use the Shell to configure the primary networking interfaces]({{< relref "ConsoleSetupMenuScale.md" >}}) to enable GUI accessibility.
+
+After booting and gaining access to the UI, go to **System Settings > General** and [upload the system config file](/scale/scaletutorials/systemsettings/general/managesysconfigscale/#uploading-the-file). This migrates your CORE settings, imports your pools, shares, etc. into SCALE.
+
+After uploading the config file, review each area of the UI previously configured in CORE to validate pools imported and settings migrated correctly. Begin with your network settings.
 
```

Comment:
```suggestion
We'll want to add a step here to create a new SCALE built-in administrator account, as it appears that uploading the CORE config file destroys the admin user created during the SCALE install process (tested with 13.0-U6.1 > 24.04-nightly build).
```

Reply:
I'll get a snippet created for this to avoid duplicate instructions.

Reply:
Added a new snippet in the next commit.